src/aap_eda/services/ruleset/activate_rulesets.py

```diff
@@ -375,6 +375,9 @@ def activate_in_k8s(
         job_name = f"activation-job-{activation_id}"
         pod_name = f"activation-pod-{activation_id}"
 
+        activation_instance.status = ActivationStatus.RUNNING
```

Comment:
It is not proper to set the RUNNING status this early nor at the end. The RUNNING status is already set at activation_kubernetes.py by watch_job_pod. It makes sense to change the activation status to RUNNING only when k8s detects pod in running state.

Reply:
@bzwei `activation_kubernetes.py` only set instance's status, not the activation.

Reply:
we should call `save_activation_and_instance` from there

Reply:
That will cause circular import errors. Will call it later when refactor k8s.